umbra-js/src/utils/utils.ts

```diff
@@ -185,6 +185,8 @@ export async function toAddress(name: string, provider: EthersProvider) {
   address = await resolveCns(name); // will never throw, but returns null on failure
   if (address) return address;
 
+  if (name.includes('.')) throw new Error('Please check if the ENS name has a valid address record.');
```

Comment:
There are other issues that could trigger this right? Such as the name not being registered at all? So maybe we should say `Please verify the name is correct, registered, has a valid address record.`